src/components/map/MapComponent.vue

```diff
@@ -183,34 +179,35 @@ travelTimesViewStore.$subscribe(async () => {
 })
 
 stationsStore.$subscribe(async () => {
-  await updateMapStyle()
-  await componentAboveMapStore.updateListLabelsStations(rennesApp)
+  await updateTraveltimeArrow(rennesApp)
+  await componentAbovemap3dStore.updateListLabelsStations(rennesApp)
+  componentAbovemap3dStore.addListenerForUpdatePositions(rennesApp)
```

Comment:
addListenerForUpdatePositions is not call when the map is mounted 
Then the update of position of custom component are not call
![image](https://user-images.githubusercontent.com/121813008/217479331-641da5ca-c060-4bb4-af12-2a0586008fe2.png)

To fix that you can add the line "componentAbovemap3dStore.addListenerForUpdatePositions(rennesApp) " at the end of onMounted function in MapComponent